proposals/nnnn-remote-control-seat.md

```diff
@@ -0,0 +1,233 @@
+| Seat Heating  level | 0-100% | Get/Set/Notification | Level of the seat heating |
+| Seat Cooling  level | 0-100% | Get/Set/Notification | Level of the seat cooling |
+| Seat Horizontal Positon | 0-100% | Get/Set/Notification | Adjust a seat forward/backward, 0 means the nearest position to the steering wheel, 100% means the furthest position from the steering wheel |
+| Seat Vertical Position | 0-100% | Get/Set/Notification | Adjust seat height (up or down) in case there is only one actutator for seat height, 0 means the lowest position, 100% means the highest position|
+| Seat-Front Vertical Position | 0-100% | Get/Set/Notification | Adjust seat front height (in case there are two actutators for seat height), 0 means the lowest position, 100% means the highest position|
+| Seat-Back Vertical Position | 0-100% | Get/Set/Notification | Adjust seat back height (in case there are two actutators for seat height), 0 means the lowest position, 100% means the highest position|
+| Seat Back Tilt Angle | 0-100% | Get/Set/Notification | Backrest recline, 0 means the angle that back top is nearest to the steering wheel, 100% means the angle that back top is furthest from the steering wheel|
+| Head Support Horizontal Positon | 0-100% | Get/Set/Notification | Adjust head support forward/backward, 0 means the nearest position to the front, 100% means the furthest position from the front |
+| Head Support Vertical Position | 0-100% | Get/Set/Notification | Adjust head support height (up or down), 0 means the lowest position, 100% means the highest position|
+| Seat Massaging Enabled | true, false | Get/Set/Notification | Indicates whether massage is enabled for a seat |
+| Massage Mode | List of Struct {MassageZone, MassageMode} | Get/Set/Notification | list of massage mode of each zone |
+| Massage Cushion Firmness | List of Struct {Cushion, 0-100%} | Get/Set/Notification | list of firmness of each massage cushion|
+| Seat memory | Struct{ id, label, action (SAVE/RESTORE/NONE)} | Get/Set/Notification | seat memory |
+
+
+#### Additions to Mobile_API
+```xml
+<enum name="ModuleType">
+  :
+  :
+  <element name="SEAT"/>
+</enum>
+
+<enum name="MassageZone">
+  <description>List possible zones of a multi-contour massage seat.</description>
+  <element name="LUMBAR">
+    <description>The back of a multi-contour massage seat. or SEAT_BACK</description>
+  </element>
+  <element name="SEAT_CUSHION">
+    <description>The bottom a multi-contour massage seat. or SEAT_BOTTOM </description>
+  </element>
+</enum>
+
+<enum name="MassageMode">
+  <description>List possible modes of a massage zone.</description>
+  <element name="OFF"/>
+  <element name="LOW"/>
+  <element name="HIGH"/>
+</enum>
+
+<struct name="MassageModeData">
+  <description>Specify the mode of a massage zone.</description>
+  <param name="massageZone" type="MassageZone" mandatory="true"></param>
+  <param name="massagegMode" type="MassageMode" mandatory="true"></param>
```

Comment:
Please change "massagegMode" to "massageMode"

Reply:
changed, thank you